src/components/auth/forgot-password.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+
+import { Formik } from "formik";
+import * as yup from "yup";
+
+import Button from "@material-ui/core/Button";
+import Box from "@material-ui/core/Box";
+
+import LoginLayout from "../common/login-layout";
+import AuthClient from "../../api/auth-client"
+import { useStyles } from './auth-style';
+
+import FormikInputField from "../UI/FormikComponents/FormikInputField"
+
+const ForgotPasswordSchema = yup.object().shape({
+  email: yup.string().email("Enter a valid email").required("Email is required"),
+});
+
+const ForgotPassword = () => {
+  const classes = useStyles();
+  const [message, setMessage] = useState("");
+
+  const getFormikHtml = () => {
+    return (<Formik
+        initialValues={{ email: "", message: "" }}
+        validationSchema={ForgotPasswordSchema}
+        onSubmit={(values, { setSubmitting, setErrors }) => {

```

Comment:
don't use inline functions.
Also you can make formikHtml and messageHtml separate components

Reply:
because  I do not see messageHtml to be used anywhere else so I left it here